app-modules/service-management/src/Filament/Blocks/ServiceRequestTypeEmailTemplateButtonBlock.php

```diff
@@ -58,6 +59,12 @@ class ServiceRequestTypeEmailTemplateButtonBlock extends TiptapBlock
     public function getFormSchema(): array
     {
         return [
+            TextInput::make('view_service_request')
```

Comment:
I would maybe call this `label`. It will then need to be changed in the blade files.